.github/workflows/sanity.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: '1.17'
+      - name: Vendor dependencies
+        run: go mod vendor
```

Comment:
We may want to also do this throughout the other workflows at some point, but this is fine for now :+1: